### ISSUE TITLE
Too much duplicate Issues

### ISSUE THREAD

**Issue by TheSpringBoy** [ottomated/CrewLink#297]
# Pre-Flight Checklist
Please use this checklist to avoid spamming:

1. [ ] I am not asking a question => use the [Discord](https://discord.gg/9mwuVNA) if you are
2. [ ] I have checked that someone else has not suggested this using the [search bar](https://github.com/ottomated/CrewLink/issues?q=is%3Aissue)
3. [ ] My feature request is not one of the **commonly suggested features**:
- Adjustable voice radius
- Mobile support
- Linux / MacOS support
- Itch.io support

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

@ottomated 
I know that you just add this cool template. but because you are the only one that can close Issues. there are more than 100, which at least 50 of them already fixed.
So I think you should create some kind of team that can close the Issues after they see that the Issue is done.
What do you think?

*And btw, you can't really tick the Checklist

**Comment by tnguyen21** [comment]
Agreed, I can help with going through tickets and trying to gather them up. Any preferred way of how we should aggregate them?

Create a new issue that captures all the duplicate issues? Close all but one of them?

**Comment by Ferrero11** [comment]
@ottomated I would love to help too on that team. This app has become really successful and widespread and I would love to help so that you can devote all your time to fixing the actual bugs and more projects. 

I like the idea tnguyen suggested but also maybe making a discord group so we don't double up capturing issues that were already put into another aggregate. I don't know if this is possible but maybe giving the team only some permission on the repository.
